2022-08-11 17:05:50,895 INFO    Thread-45 :882448 [wandb_setup.py:_flush():76] Configure stats pid to 882448
2022-08-11 17:05:50,896 INFO    Thread-45 :882448 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-11 17:05:50,896 INFO    Thread-45 :882448 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-11 17:05:50,896 INFO    Thread-45 :882448 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_c', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': '4fwlnyxm', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-s074ty2z/config-4fwlnyxm.yaml', 'sweep_id': 's074ty2z'}
2022-08-11 17:05:50,896 WARNING Thread-45 :882448 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-11 17:05:50,896 INFO    Thread-45 :882448 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-11 17:05:50,896 INFO    Thread-45 :882448 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220811_170550-4fwlnyxm/logs/debug.log
2022-08-11 17:05:50,896 INFO    Thread-45 :882448 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220811_170550-4fwlnyxm/logs/debug-internal.log
2022-08-11 17:05:50,900 INFO    Thread-45 :882448 [wandb_init.py:init():495] calling init triggers
2022-08-11 17:05:50,900 INFO    Thread-45 :882448 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 8, 'iteration': 300, 'learning_rate': 0.0009873035501613558, 'model': 'COCO-Detection/faster_rcnn_R_101_FPN_3x.yaml'}
config: {}
2022-08-11 17:05:50,903 INFO    Thread-45 :882448 [wandb_init.py:init():548] starting backend
2022-08-11 17:05:50,903 INFO    Thread-45 :882448 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-11 17:05:50,913 INFO    Thread-45 :882448 [backend.py:ensure_launched():217] starting backend process...
2022-08-11 17:05:50,947 INFO    Thread-45 :882448 [backend.py:ensure_launched():222] started backend process with pid: 919515
2022-08-11 17:05:50,948 INFO    Thread-45 :882448 [wandb_init.py:init():558] backend started and connected
2022-08-11 17:05:50,963 INFO    Thread-45 :882448 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 8, 'iteration': 300, 'learning_rate': 0.0009873035501613558, 'model': 'COCO-Detection/faster_rcnn_R_101_FPN_3x.yaml'}
2022-08-11 17:05:50,967 INFO    Thread-45 :882448 [wandb_init.py:init():634] updated telemetry
2022-08-11 17:05:50,969 INFO    Thread-45 :882448 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-11 17:05:51,760 INFO    Thread-45 :882448 [wandb_run.py:_on_init():1919] communicating current version
2022-08-11 17:05:51,802 INFO    Thread-45 :882448 [wandb_run.py:_on_init():1923] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-11 17:05:51,802 INFO    Thread-45 :882448 [wandb_init.py:init():700] starting run threads in backend
2022-08-11 17:05:52,886 INFO    Thread-45 :882448 [wandb_run.py:_console_start():1893] atexit reg
2022-08-11 17:05:52,890 INFO    Thread-45 :882448 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-11 17:05:52,890 INFO    Thread-45 :882448 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-11 17:05:52,890 ERROR   Thread-45 :882448 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-11 17:05:52,891 INFO    Thread-45 :882448 [wandb_init.py:init():732] run started, returning control to user process
2022-08-11 17:06:02,909 INFO    Thread-45 :882448 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_c/4fwlnyxm
2022-08-11 17:06:02,910 INFO    Thread-45 :882448 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-08-11 17:06:02,915 INFO    Thread-45 :882448 [wandb_run.py:_restore():1834] restore
2022-08-11 17:06:02,919 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-11 17:06:03,263 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-11 17:06:03,787 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5310
}

2022-08-11 17:06:03,889 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5951
}

2022-08-11 17:06:03,992 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5951
}

2022-08-11 17:06:04,094 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5951
}

2022-08-11 17:06:04,197 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-11 17:06:04,300 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-11 17:06:04,402 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-11 17:06:04,505 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-11 17:06:04,607 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-11 17:06:04,709 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-11 17:06:05,451 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-11 17:06:05,991 INFO    Thread-45 :882448 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}
local_info {
}

2022-08-11 17:06:07,124 INFO    Thread-45 :882448 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-11 17:06:07,125 INFO    Thread-45 :882448 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-11 17:06:07,130 INFO    Thread-45 :882448 [wandb_run.py:_footer_sync_info():3055] logging synced files
